posthog-android/lint-baseline.xml

```diff
@@ -4,8 +4,8 @@
     <issue
         id="GradleDependency"
         message="A newer version of androidx.lifecycle:lifecycle-process than 2.6.2 is available: 2.8.7"
-        errorLine1="    implementation(&quot;androidx.lifecycle:lifecycle-process:${PosthogBuildConfig.Dependencies.LIFECYCLE}&quot;)"
-        errorLine2="                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~">
```

Comment:
These changes fixed `make compile` locally. I didn't want to change dependencies or anything as I don't have context. I don't think my changes introduced these build errors.

Reply:
if you run `make generateLintBaseLine`, this will generate the new file with the latest versions, we don't want to depend on the latest version to avoid breaking changes.

Reply:
@marandaneto thanks for the tip!